Neos.Flow/Classes/Mvc/Routing/Dto/Action.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+namespace Neos\Flow\Mvc\Routing\Dto;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Mvc\ActionRequest;
+
+/**
+ * @Flow\Proxy(false)
+ */
+final class Action
+{
+    private function __construct(
+        private string $actionName,
+        private string $controllerName,
+        private string $packageKey,
+        /** @deprecated with Flow 8.0 - The use of subpackage keys is discouraged and only supported for backwards compatibility */
+        private ?string $subpackageKey,
+        private string $format,
+        private array $additionalArguments,
```

Comment:
Why additionalArguments and not arguments

Reply:
technically because `@action`, `@controller`, etc. are _all_ arguments. But from this DTO point of view, they are just arguments, I agree!